lib/config/migrations/custom/recreate-closed-migration.ts

```diff
@@ -0,0 +1,12 @@
+import is from '@sindresorhus/is';
+import { AbstractMigration } from '../base/abstract-migration';
+
+export class RecreateClosedMigration extends AbstractMigration {
+  override readonly propertyName = 'recreateClosed';
+
+  override run(value: unknown): void {
+    if (is.boolean(value)) {
+      this.rewrite(value ? 'always' : 'auto');
```

Comment:
are we sure `value !== true` should be `auto`? `false` would be `auto` too. 🤔

Reply:
We check for boolean and do the following mapping:
false -> auto
true -> always